dynamic_forms/static/dynamic_forms.js

```diff
@@ -61,10 +61,21 @@ function dynamic_forms_textarea() {
     var turn_edit_off = function() {
       textbox.hide();
       spanbox.show();
+      menu.removeClass('hidden');
     };
+    var turn_preview_on = function() {
+      base.hide();
+      base.after('<span class="textarea preview">' + base.val() + '</span>');
+    }
+    var turn_preview_off = function() {
+      textbox.find('.preview').remove();
+      base.show();
```

Comment:
I realized when testing the changes that when the preview is turned off, the focus isn't returned to the textbox (and the previously selected selection), so I'll fix that as well. (And also change that when the _preview_ button is pressed, the focus is placed on the _unpreview_ button, so it's easier to toggle.)